### ISSUE TITLE
ANSI sequences work for colours but not bold/italic, underline etc

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#144275]
Originally reported in https://github.com/dotnet/interactive/issues/1839, outputs from below powershell code in .NET Interactive Notebook are not rendered properly

```
$psstyle.Foreground.Red + "Hello" + $psstyle.Reset
$psstyle.Bold + "Hello" + $psstyle.Reset
$psstyle.Italic + "Hello" + $psstyle.Reset
$psstyle.Reverse + "Hello" + $psstyle.Reset
$psstyle.Strikethrough + "Hello" + $psstyle.Reset
$psstyle.Underline + "Hello" + $psstyle.Reset
$psstyle.Background.BrightWhite + $psstyle.Foreground.Black + "Hello" + $psstyle.Reset
```

**Comment by rebornix** [comment]
Fixed via 4a6763d8d52f22704450ffa358c0ed0ccfdac4ab

**Comment by rchiodo** [comment]
![image](https://user-images.githubusercontent.com/19672699/160028667-fdf828eb-20a3-4e98-832b-b34902865281.png)
